<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelNome">
       <property name="text">
        <string>&amp;Nome:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditNome</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNome"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBoxSexo">
         <property name="title">
          <string>Sexo:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioBtnFeminino">
            <property name="text">
             <string>Feminino</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioBtnMasculino">
            <property name="text">
             <string>Masculino</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>radioBtnFeminino</zorder>
         <zorder>radioBtnMasculino</zorder>
         <zorder>lineEditNome</zorder>
         <zorder>labelNome</zorder>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxEMail">
         <property name="text">
          <string>&amp;recebe e-mail</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelNome_2">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>UF:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="fontComboBox">
           <property name="currentText">
            <string>SP</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxEstadoCivil">
       <property name="title">
        <string>Estado civil:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioBtnSolteiro">
          <property name="text">
           <string>Solteiro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioBtnCasado">
          <property name="text">
           <string>Casado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioBtnDivorciado">
          <property name="text">
           <string>Divorciado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioBtnViuvo">
          <property name="text">
           <string>Viuvo</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushBtnOK">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>labelNome</zorder>
  <zorder>textEditName</zorder>
  <zorder>pushBtnOK</zorder>
  <zorder>groupBoxSexo</zorder>
  <zorder>groupBoxEstadoCivil</zorder>
  <zorder>pushBtnCancel</zorder>
  <zorder>lineEditNome</zorder>
  <zorder>labelUF</zorder>
  <zorder>checkBoxEMail</zorder>
  <zorder>fontComboBoxUF</zorder>
  <zorder>fontComboBoxUF</zorder>
  <zorder>labelUF</zorder>
  <zorder>checkBoxEMail</zorder>
  <zorder>fontComboBoxUF</zorder>
  <zorder>fontComboBoxUF</zorder>
  <zorder>fontComboBoxUF</zorder>
  <zorder>labelUF</zorder>
  <zorder>fontComboBox</zorder>
  <zorder>labelNome_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditNome</tabstop>
  <tabstop>radioBtnFeminino</tabstop>
  <tabstop>radioBtnMasculino</tabstop>
  <tabstop>checkBoxEMail</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>radioBtnSolteiro</tabstop>
  <tabstop>radioBtnCasado</tabstop>
  <tabstop>radioBtnDivorciado</tabstop>
  <tabstop>radioBtnViuvo</tabstop>
  <tabstop>pushBtnOK</tabstop>
  <tabstop>pushBtnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
